sequencer/config.go

```diff
@@ -73,3 +74,23 @@ func SetTMConfig(rlpCfg config.RollappConfig) error {
 	tomlCfg.Set("rpc.cors_allowed_origins", []string{"*"})
 	return utils.WriteTomlTreeToFile(tomlCfg, configFilePath)
 }
+
+func GetRPCPort(rlpCfg config.RollappConfig) (string, error) {
```

Comment:
also would suggest to make a generic function as the first lines will be repetitive for each config you're gonna fetch.